pkg/machines/libvirt-common.es6

```diff
@@ -186,6 +187,17 @@ export function parseDumpxml(dispatch, connectionName, domXml, id_overwrite) {
 
     const currentMemoryUnit = currentMemoryElem.getAttribute("unit");
     const currentMemory = convertToUnit(currentMemoryElem.childNodes[0].nodeValue, currentMemoryUnit, units.KiB);
+    let maxMemoryTmp;
+    if (maxMemoryElem) {
+        const maxMemoryUnit = maxMemoryElem.getAttribute("unit");
+        maxMemoryTmp = convertToUnit(maxMemoryElem.childNodes[0].nodeValue, maxMemoryUnit, units.MiB);
+    } else {
```

Comment:
I don't this this is what you wanted to do here ;) If we get here it means that memory element does not exist in the XML, no need to recheck it.